gwlfe/CalcCnErosRunoffSed.py

```diff
@@ -27,6 +27,9 @@ def CalcCN(z, i, Y, j):
     z.UncontrolledQ = 0
     z.RetentionEff = 0
 
+    if not hasattr(z, 'AdjUrbanQTotal'):
+        z.AdjUrbanQTotal = 0
```

Comment:
Sounds good. I just moved it and the tests and the Royal Palm gms file both still worked.